partition_processor.go

```diff
@@ -71,7 +74,6 @@ type PartitionProcessor struct {
 	// runnerErrors store the errors occuring during runtime of the
 	// partition processor. It is created in Setup and after the runnerGroup
 	// finishes.
-	runnerErrors chan error
```

Comment:
we should also remove the comment

Reply:
damn, that happens if you edit your code in an external diff-editor, not the actual ide :facepalm: